instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/test/opmsg_deployment.go

```diff
@@ -0,0 +1,234 @@
+// Copied from https://github.com/mongodb/mongo-go-driver/blob/d46f29c34e9ff3968a5c00dcdea8fb41a20a2b2f/mongo/integration/mtest/opmsg_deployment.go
```

Comment:
The [precommit target for checking licenses](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/9ddea00fc1eaca6c408db9fa4232831730a6c640/Makefile#L189) doesn't appear to account for the case where code is copied from a third party. Should `license-check` be extended to ignore files with a third party copyright?

Reply:
For now I've extended the`license-check` make target and codecov.yaml file to account for this case. See comment concerning mtest

Reply:
The OTel license header needs to added to the top of this file. The existing licences header needs to remain below the added OTel license header and the copied comment can also be added below it.

See [this](https://github.com/open-telemetry/opentelemetry-go/blob/6d45f283c7abf2ee5284c769f9ae7109096d3564/sdk/log/ring.go#L1-L16) for an example.

Reply:
More: https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md#code-attribution

Reply:
@MrAlias @pellared @dmathieu Quick update: we are still awaiting feedback concerning the licensing on our end. For simplicity, it may be simpler to just the `mtest` solution. Apologies for the delay on this.